components/renku_data_services/platform/db.py

```diff
@@ -2,12 +2,17 @@
 
 from collections.abc import Callable
 
-from sqlalchemy import select
+from sqlalchemy import func, select
 from sqlalchemy.ext.asyncio import AsyncSession
 
 from renku_data_services import base_models, errors
+from renku_data_services.authz.authz import Authz
+from renku_data_services.base_api.pagination import PaginationRequest
 from renku_data_services.platform import models
 from renku_data_services.platform import orm as schemas
+from renku_data_services.project.db import ProjectRepository
+
+# from renku_data_services.utils.core import with_db_transaction
```

Comment:
question: leftover from other code? If not needed we should fully remove it.

Reply:
Fixed in 4e2dd792d47e41c892c74dac61d7ef90940a9bac